src/test/java/core/basesyntax/service/impl/FruitRecordParserImplTest.java

```diff
@@ -0,0 +1,97 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.dto.FruitRecordDto;
+import core.basesyntax.service.FruitRecordParser;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitRecordParserImplTest {
+    private static FruitRecordDto fruitRecordDtoBalance;
+    private static FruitRecordDto fruitRecordDtoSupply;
+    private static FruitRecordDto fruitRecordDtoReturn;
+    private static FruitRecordDto fruitRecordDtoPurchase;
+    private static FruitRecordParser fruitRecordParser;
+    private final List<String> strings = new ArrayList<>();
+    private final List<FruitRecordDto> expected = new ArrayList<>();
+
+    @BeforeClass
+    public static void beforeClass() {
+        fruitRecordDtoBalance = new FruitRecordDto(OperationType.BALANCE, "banana", 100);
+        fruitRecordDtoSupply = new FruitRecordDto(OperationType.SUPPLY, "apple", 125);
+        fruitRecordDtoReturn = new FruitRecordDto(OperationType.RETURN, "banana", 150);
+        fruitRecordDtoPurchase = new FruitRecordDto(OperationType.PURCHASE, "apple", 75);
+        fruitRecordParser = new FruitRecordParserImpl();
+    }
+
+    @Before
+    public void setUp() {
```

Comment:
it's better to add After where you will clear expected and strings, this way you will get more predictable behaviour

Reply:
done